### ISSUE TITLE
Questions about reproducing experimental results

### ISSUE THREAD

**Issue by zaixizhang** [snap-stanford/pretrain-gnns#33]
Hi,

I find your work very interesting and tried to reproduce some of the results with your code. However, I find some of my results are inconsistent with Table 1. For example, for the Clintox dataset, attributemasking has ROC-AUC 82% while supervised + masking has ROC-AUC 78%. These results are inconsistent or may be contrary to the results in the paper. 

I use the scaffold data splitting and finetune 100 epochs.

**Comment by weihua916** [comment]
Hi! Interesting. Just to confirm: you were exactly following the instruction in [README.md](https://github.com/snap-stanford/pretrain-gnns#reproducing-results-in-the-paper) and could not reproduce the results, right? If so, could you share a bit more context, probably the package version etc?
